### DISCUSSION TITLE
Inference Engine

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#14] [Q&A] [answered]
What is backward chaining?

**Comment by GouniSahithi** [comment, marked as answer]
Backward chaining is also known as backward deduction or backward reasoning method when using an inference engine.
It is a form of reasoning,which starts with goal and works backward to determine what facts must be asserted so that goal can be achieved.
It is top-down approach.
It is called a goal-driven approach,as a list of goals decides which rules are selected and used.
It is used in game theory,automated theorem proving tools, inference engine,....
